scripts/xcode_select_current_version.sh

```diff
@@ -3,7 +3,7 @@
 if [ -n "$XCODE_PATH_OVERRIDE" ]; then # If someone calls this with the XCODE_PATH_OVERRIDE variable set to a path to a developer dir, use it instead
     XCODE_PATH="$XCODE_PATH_OVERRIDE"
 else
-    XCODE_PATH='/Applications/Xcode_12.5.app/Contents/Developer'
```

Comment:
It seems that Xcode 12.4 is the default for the macos-10.5 image.

https://github.com/actions/virtual-environments/blob/main/images/macos/macos-10.15-Readme.md